module_utils/ns1.py

```diff
@@ -25,7 +25,11 @@ def missing_required_lib(msg, reason=None, url=None):
     HAS_NS1 = False
 
 
-NS1_COMMON_ARGS = dict(apiKey=dict(required=True, no_log=True))
+NS1_COMMON_ARGS = dict(
+    apiKey=dict(required=True, no_log=True),
+    endpoint=dict(required=False, type='str', default=None),
+    ignore_ssl=dict(required=False, type='bool', default=None)
+)
```

Comment:
is using `dict()` instead of literals specifically intended or required? Literals are definitely preferred in the community, and by me (more readable but they are also faster) I feel like there's some words to this effect in a PEP somewhere (prefer literals to e.g. list() or dict()) but 1 min of googling didn't find it for me.
Also not gonna fuss about changing it if you agree but the rest of the codebase is full of this style.

Reply:
Using the dict constructor vs the literal is specifically intended.  It's the style used by all modules in Ansible core (where this code eventually needs to be merged).  Check out some [other modules](https://github.com/ansible/ansible/tree/devel/lib/ansible/modules) to get a sense of that project's code style.